File: api/Infrastructure/Entities/Team.cs
namespace api.Infrastructure.Entities;
public class Team
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public int CreatedById { get; set; }
    public DateTimeOffset CreatedDate { get; set; }

    public Team(
        string name,
        string description,
        int createdById)
    {
        Name = name;
        Description = description;
        CreatedById = createdById;
    }
}

File: ui/src/models/Team.ts
export default class {
    id: number;
    name: string;
    description: string;
    createdById: number;
    createdDate: Date;

    constructor(
        id: number,
        name: string,
        description: string,
        createdById: number,
        createdDate: Date,
    ) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.createdById = createdById;
        this.createdDate = createdDate;
    }
}
